docs/backend-system/architecture/06-modules.md

```diff
@@ -6,35 +6,42 @@ sidebar_label: Modules
 description: Modules for backend plugins
 ---
 
-## Creating Modules
+Backend modules are used to extend plugins with additional features or change existing behavior. The must always be installed in the same backend instance as the plugin that they extend, and may only extend a single plugin. Modules interact with their target plugin using the extension points registered by the plugin, while also being able to depend on the services of that plugin.
```

Comment:
```suggestion
Backend modules are used to extend plugins with additional features or change existing behavior. They must always be installed in the same backend instance as the plugin that they extend, and may only extend a single plugin. Modules interact with their target plugin using the extension points registered by the plugin, while also being able to depend on the services of that plugin.
```

probably make the first occurrences of "extension point" and "plugin" into links to those articles too. Cross linking like that can be powerful